<launch>
  <!-- <arg name="model" default="$(find open_critter)/xacro/critter.xacro"/>
  <param name="robot_description" command="$(find xacro)/xacro $(arg model)" />
  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" /> -->
  <node name="robot_bridge"  pkg="open_critter" type="robot_bridge">
    <param name="namespace"  value="open_critter"/>
    <param name="hostname"   value="zero"/>
    <param name="portnumber" value="9001"/>
  </node>
</launch>
